scripts/build-logos.js

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env node
+
+const productLogos = require('../public/data/product-logos.json');
+const { loadConfig, optimize } = require('svgo');
+const { appendFile, mkdir, readdir, readFile, writeFile, copyFile } = require('fs').promises;
+const camelcase = require('camelcase');
+const path = require('path');
+const babel = require('@babel/core');
+const { transform } = require('@svgr/core');
+const pluginTransformReactJsx = require('@babel/plugin-transform-react-jsx');
+
+const OUTPUT_DIR = path.join(__dirname, '../generated/logos');
```

Comment:
Any reason we have `generated` dir instead of `dist`? 🤔  If not, can we switch it back?

Reply:
Yes, the generated folder is built before compiling, since importing from dist doesnt work (chicken and egg problem) so we get compile errors. The `generated` folder does not contain files that we should toch at all, since everything is generated automatically. The `public` folder is the opposite, all files in there are manually maintained.